contrib/containers/ci/Dockerfile

```diff
@@ -37,14 +28,11 @@ RUN groupadd -g ${GROUP_ID} dash
 RUN useradd -u ${USER_ID} -g dash -s /bin/bash -m -d /dash dash
 
 # Packages needed for all target builds
-RUN dpkg --add-architecture i386
-RUN apt-get update && apt-get install $APT_ARGS g++-9-multilib && rm -rf /var/lib/apt/lists/*
-RUN apt-get update && apt-get install $APT_ARGS g++-arm-linux-gnueabihf && rm -rf /var/lib/apt/lists/*
-RUN apt-get update && apt-get install $APT_ARGS g++-mingw-w64-x86-64 && rm -rf /var/lib/apt/lists/*
-RUN apt-get update && apt-get install $APT_ARGS wine-stable wine32 wine64 bc nsis xorriso libncurses5 && rm -rf /var/lib/apt/lists/*
-RUN apt-get update && apt-get install $APT_ARGS python3-zmq && rm -rf /var/lib/apt/lists/*
-RUN apt-get update && apt-get install $APT_ARGS gawk jq && rm -rf /var/lib/apt/lists/*
-RUN apt-get update && apt-get install $APT_ARGS imagemagick libcap-dev librsvg2-bin libz-dev libbz2-dev libtiff-tools && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install $APT_ARGS g++-9-multilib g++-arm-linux-gnueabihf g++-mingw-w64-x86-64       \
```

Comment:
* You can drop the `apt-get update` because of the earlier `dpkg --add-architecture i386` call

* The package manager calls can be perhaps been formatted like:

  ```
  apt-get install $APT_ARGS \
  g++-9-multilib \
  g++-arm-linux-gnueabihf \
  g++-mingw-w64-x86-64 \
  [...]
  ```

Reply:
No, any time we call an apt install, we must do an apt update so we don't have out of date packages from a prior cache, plus apt update is fast.

I don't have a strong formatting opinion, so I'm not gonna do it atm, but feel free to do it yourself.